packages/legend-studio/src/stores/editor-state/element-editor-state/service/ServiceRegistrationState.ts

```diff
@@ -217,19 +231,42 @@ export class ServiceRegistrationState {
           versionInput,
         )) as ServiceRegistrationResult;
       if (this.activatePostRegistration) {
+        this.setRegistrationPhase(
+          SERVICE_REGISTRATION_PHASE.ACTIVATING_SERVICE,
+        );
         yield this.editorStore.graphManagerState.graphManager.activateService(
           serverUrl,
           serviceRegistrationResult.serviceInstanceId,
         );
       }
-      this.setShowModal(false);
-      this.editorStore.applicationStore.notifySuccess(
-        `service with patten ${serviceRegistrationResult.pattern} registered ${
-          this.activatePostRegistration ? ' and activated ' : ''
-        } on ${serviceRegistrationResult.serverURL}`,
-        undefined,
-        null,
-      );
+      const message = `Service with patten ${
+        serviceRegistrationResult.pattern
+      } registered ${this.activatePostRegistration ? 'and activated ' : ''}on ${
+        serviceRegistrationResult.serverURL + serviceRegistrationResult.pattern
+      }.`;
+      this.editorStore.setActionAltertInfo({
+        message,
+        prompt: 'You can open the corresponding service page',
+        type: ActionAlertType.STANDARD,
+        onEnter: (): void => this.editorStore.setBlockGlobalHotkeys(true),
+        onClose: (): void => this.editorStore.setBlockGlobalHotkeys(false),
+        actions: [
+          {
+            label: 'Open Service Page',
+            type: ActionAlertActionType.PROCEED,
+            handler: (): void => {
+              this.editorStore.applicationStore.navigator.openNewWindow(
+                `${config.metadataUrl}${serviceRegistrationResult.pattern}`,
```

Comment:
why is it `metadataUrl` and not the service registration realm?

Reply:
you wanna open it to the service GUI where it has all the metadata about the service. there they can choose to execute if they want.